### DISCUSSION TITLE
Separate duplicate data from large data execl sheet

### DISCUSSION THREAD

**Discussion by priyadarshu** [EddieHubCommunity/support#1177] [Tips and Tricks]
One way is to follow this article by Microsoft - https://support.microsoft.com/en-us/office/find-and-remove-duplicates-00e35bea-b46a-4d5d-b28e-66a552dc138d

Second way is  to write Excel VBA Macros to loop through the columns and remove duplicates

https://www.automateexcel.com/vba/removing-duplicate-values-in-excel-vba/

**Comment by hasthamalp** [comment]
THANK YOU @priyadarshu
